### ISSUE TITLE
Where am I supposed to find "mqtt-wrapper.h"

### ISSUE THREAD

**Issue by GearDownForWhat** [i3detroit/mqtt-J1708-LED-sign#1]
Also, I'm curious if this is for Sunrise systems  Model NXTP7X962M

**Comment by mtfurlan** [comment]
I don't know what the model is, this is everything I know: https://www.i3detroit.org/wiki/LED_Matrix_Display
I'll ask someone to look a bit closer for a model though, but it isn't clearly marked.

The missing files are in https://github.com/i3detroit/mqtt-wrapper
It's not pretending to be a finished library, it's a way for me to hide some boilerplate. 
I do not promise backwards compatibility, and it's possible you will have to look at commit dates to get the right versions, sorry about that.

I'm working on the build scripts to autobuild a lot of this, and I will be revisiting that library soonish, and hopefully it will hit a state I'm happy with and won't change much.

**Comment by GearDownForWhat** [comment]
Its looks similar to the one I have, but defiantly not the same, mine has an aluminum frame.

I’m a little confused, what is the password and hostname for? Is this some wifi setup that you have controlling the Arduino?

**Comment by mtfurlan** [comment]
Yeah, we're running a mqtt broker that devices connect to over wifi. If you just want sample code, look at https://github.com/i3detroit/mqtt-J1708-LED-sign/blob/master/referenceCode.py that's what I build the arduino version off of.

**Comment by mtfurlan** [comment]
It is a Sunrise systems Inc NXTP7X962M/J1708.

**Comment by GearDownForWhat** [comment]
What are you controlling it with? What kind of serial device?

**Comment by mtfurlan** [comment]
Right now it's an esp, but I had it working with an ftdi chip. I think.

**Comment by abzman** [comment]
It works with a stack of: sign - rs232 to rs485 converter - rs232 to logic level serial converter - esp8266 or FTDI usb to logic level serial chip
